.github/workflows/ruby.yml

```diff
@@ -35,6 +35,8 @@ jobs:
         ruby-version: ['2.7', '3.0']
 
     runs-on: ${{ matrix.os }}
+    env:
+      BUNDLE_WITHOUT: typecheck
```

Comment:
* should we keep typechecking for 3.0?
* if we don't enable typecheck for 3.0 it looks like we could add windows-latest to list of OSes and get rid of test-windows

Reply:
Consolidated everything into one job, and CI is happy.  Good call, thank you for suggesting it.